llarp/ev.hpp

```diff
@@ -25,6 +25,7 @@ struct asio_evt_pkt
   size_t sz;  // if this doesn't match what is in the packet, note the error
 };
 #else
+#include <sys/event.h>
```

Comment:
oh you forgot the CPP includes
```c
#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) \
    || (__APPLE__ && __MACH__)
#include <sys/event.h>
#endif
```
since `sys/event.h` does not exist on linux (nor do we expect linux users to have `libkqueue` installed, hence the `epoll(2)` implementation of event loop)

Reply:
done!